pulp_container/app/registry_api.py

```diff
@@ -292,7 +292,12 @@ def default_response_headers(self):
         Provide common headers to all responses.
         """
         headers = super().default_response_headers
-        headers.update({"Docker-Distribution-Api-Version": "registry/2.0"})
+        headers.update(
+            {
+                "Docker-Distribution-Api-Version": "registry/2.0",
+                "X-Registry-Supports-Signatures": "1",
```

Comment:
This header name is already defined in stages file https://github.com/pulp/pulp_container/blob/signing/pulp_container/app/tasks/sync_stages.py#L43, how about putting it into the constants and using it here and there?